cypress/integration/e2e/table/undo-redo.spec.ts

```diff
@@ -0,0 +1,72 @@
+/// <reference path="../../../support/index.d.ts" />
```

Comment:
Czy `reference path` jest tutaj potrzebny?

https://github.com/Project-Summer-AI-Lab-Glider/nurse-scheduling-problem-frontend/pull/127/files#r547565755

Reply:
Ten niestety tak. Referencje, której się pozbyłem odnoisły się do modułów z z zainstalowanych przez nas bibliotek(cypress i cypress-file-upload). W takim przypadku można ją zaciągnąć raz w pliku [tsconfig.json](https://github.com/Project-Summer-AI-Lab-Glider/nurse-scheduling-problem-frontend/blob/e5bfd2c9ba0e30b615227cb60824b5355b63706c/cypress/tsconfig.json). 
Niestety w tym przypadku mi się nie udało tego zrobić, ponieważ tutaj mamy do czynienia z naszym customym modułem. Chwilę probowałem, ale z marnym skutkiem i to tak juz to zostawiłem.

Reply:
Jak dla mnie to żaden problem, ale myślałem że zapomniałeś to usunąć, więc zapytałem